refs/types.proto

```diff
@@ -77,6 +77,18 @@ message OwnerID {
   bytes value = 1 [json_name = "value"];
 }
 
+// NeoFS subnetwork identifier.
+//
+// `SubnetID` is a 4-byte long.
+//
+// String representation is base-10 integer value.
+//
+// JSON representation is a JSON number.
+message SubnetID {
+  // Integer subnetwork identifier.
+  fixed32 value = 1 [json_name = "value"];
```

Comment:
Let's clearly indicate that zero identifies the zero subnet.

Reply:
This is more about the `PlacementPolicy` interpretatiion than the id itself.

Reply:
> why have you chosen `fixed32`?

`fixed32` was mentioned for in-place ID checks in smart contract. With `uint32` type ID will be encoded as varint, which is much harder to decode.